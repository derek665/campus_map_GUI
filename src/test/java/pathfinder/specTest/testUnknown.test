# test correct ouput when nodes are not defined

CreateGraph testNone

FindPath testNone A B