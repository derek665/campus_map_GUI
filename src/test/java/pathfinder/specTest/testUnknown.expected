# test correct ouput when nodes are not defined

created graph testNone

unknown node A
unknown node B